src/helpers/createEmailBadge.ts

```diff
@@ -11,9 +11,14 @@ function createContract(provider: Web3Provider) {
   )
 }
 
-export default async function (provider: Web3Provider, proof: EmailProof) {
+export default async function (
+  provider: Web3Provider,
+  proof: EmailProof,
+  maxFeePerGas?: string
+) {
+  if (!proof.result) throw new Error('Invalid proof')
```

Comment:
Looks like this is the same, as in `createExternalERC721Badge.ts`, maybe we should refactor it?